k8s-install/scripts/install-cni.sh

```diff
@@ -136,7 +140,11 @@ echo "CNI config: $(cat ${TMP_CONF})"
 sed -i s/__SERVICEACCOUNT_TOKEN__/${SERVICEACCOUNT_TOKEN:-}/g $TMP_CONF
 
 # Move the temporary CNI config into place.
-mv $TMP_CONF /host/etc/cni/net.d/${FILENAME}
+if ! mv $TMP_CONF /host/etc/cni/net.d/${FILENAME}
```

Comment:
Wonder, can we just `set -e` at the top of the script and then we won't need the special case handling here? It will also ensure we catch all errors in the script.

What's not clear to me is if there are errors we're OK with, if they occur.  In which case we'd need to explicitly add exceptions for them.

Reply:
I was considering the same, but thought we had planned for some potential errors, I'll tinker with it and if we can just add `set -e` I'll do so.